cylc/flow/cfgspec/globalcfg.py

```diff
@@ -768,7 +769,7 @@ def get_version_hierarchy(version: str) -> List[str]:
         ['', '8', '8.0', '8.0.1', '8.0.1a2', '8.0.1a2.dev']
 
     """
-    smart_ver = parse_version(version)
+    smart_ver: Version = parse_version(version)
```

Comment:
MyPy had issues here, had to set the type to keep it happy.

Reply:
```suggestion
    smart_ver: Any = parse_version(version)
```